src/test/java/com/puppycrawl/tools/checkstyle/utils/CommonUtilTest.java

```diff
@@ -440,6 +452,44 @@ public void testIsIntNull() {
             CommonUtil.isInt(null));
     }
 
+    @Test
+    public void testGetUriByFilenameFindsAbsoluteResourceOnClasspath() throws Exception {
+        final String filename =
+            "/" + getPackageLocation() + "/InputCommonUtilTest_empty_checks.xml";
+        final URI uri = CommonUtil.getUriByFilename(filename);
+        assertThat("URI is null for: " + filename, uri, is(not(nullValue())));
+    }
+
+    @Test
+    public void testGetUriByFilenameFindsRelativeResourceOnClasspath() throws Exception {
+        final String filename =
+            getPackageLocation() + "/InputCommonUtilTest_empty_checks.xml";
+        final URI uri = CommonUtil.getUriByFilename(filename);
+        assertThat("URI is null for: " + filename, uri, is(not(nullValue())));
+    }
+
+    @Test
+    public void testGetUriByFilenameFindsResourceRelativeToRootClasspath() throws Exception {
+        final String filename = "commonutil/InputCommonUtilTest_resource.txt";
+        final URI uri = CommonUtil.getUriByFilename(filename);
+        assertThat("URI is null for: " + filename, uri, is(not(nullValue())));
+        final String uriRelativeToPackage =
+            "com/puppycrawl/tools/checkstyle/utils/commonutil/InputCommonUtilTest_resource.txt";
+        assertThat("URI is relative to package " + uriRelativeToPackage,
+            uri.toString(), not(containsString(uriRelativeToPackage)));
+        final ByteArrayOutputStream out = new ByteArrayOutputStream();
+        try (InputStream in = uri.toURL().openStream()) {
+            IOUtils.copy(in, out);
+        }
+        assertThat("Content mismatches for: " + uri,
+            new String(out.toByteArray(), StandardCharsets.UTF_8), startsWith("good"));
+    }
+
+    @Override
+    protected String getPackageLocation() {
```

Comment:
please move this method to the top of class, right after fields.

Reply:
It better to reply "done" when you finished review item.

Reply:
OK - I'll do that.
I expected the button "Resolve conversation" to do more or less the same, but apparently not (not sure what it actually does then).

Reply:
Done.

Reply:
> I expected the button "Resolve conversation" to do more or less the same, but apparently not 

We don't get notifications on clicking resolve and pushing commits, so it could go unnotice that you made changes. Also we ask that we be the ones to click resolve conversation so we can use it as a way to double verify the work.